at-csv/src/main/java/io/engineblock/activities/csv/FileActivity.java

```diff
@@ -24,23 +26,43 @@
 public class FileActivity extends SimpleActivity implements ActivityDefObserver {
     private final static Logger logger = LoggerFactory.getLogger(FileActivity.class);
     private final FileStmtDocList stmtDocList;
+    private final Boolean showstmts;
     private ReadyFileStatementsTemplate readyFileStatementsTemplate;
 
     public Timer bindTimer;
```

Comment:
Can you do a clean-up pass against the code to remove unused things that might cause confusion?